starboard/tools/api_leak_detector/api_leak_detector.py

```diff
@@ -395,9 +410,20 @@ def main():
   def IsSbSymbol(symbol):
     return symbol.startswith('Sb') or symbol.startswith('kSb')
 
+  def IsAllowedPosixSymbol(symbol, sb_api_version: int):
+    if sb_api_version == 16:
+      return symbol in _ALLOWED_SB16_POSIX_SYMBOLS
+    else:
+      return False
+
+  def IsAllowedSymbol(symbol):
+    return symbol in allowed_c99_symbols or IsSbSymbol(
```

Comment:
I think this should be made simpler by separate if statements with early returns instead of one large boolean expression